src/DotnetWebApiBench.DataAccess/NorthwindDatabaseContext.cs

```diff
@@ -60,7 +71,14 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-                optionsBuilder.UseSqlite("Data Source=.\\Northwind_small.db;");
+                if (serverType != null && serverType.Equals("SQLServer", StringComparison.InvariantCultureIgnoreCase))
+                {
+                    optionsBuilder.UseSqlServer("Data Source=192.168.10.50;initial catalog=test;user id=developer;password=1234;");
```

Comment:
It is better for the default connection string to point to localhost.

Reply:
changes applied